test/jdk/java/util/TimeZone/DefaultTimeZoneTest.java

```diff
@@ -23,86 +23,96 @@
 
 /*
  * @test
- * @bug 4296930 5033603 7092679
- * @summary Make sure that Java runtime detects the platform time zone
- * correctly. Also make sure that the system time zone detection code
- * detects the "Automatically adjust clock for daylight saving
- * changes" setting correctly on Windows.
- * @run applet/manual=yesno DefaultTimeZoneTest.html
+ * @bug 4296930 5033603 7092679 8340326
+ * @summary Ensure that Java detects the platform time zone correctly, even
+ * if changed during runtime. Also ensure that the system time zone detection code
+ * detects the "Automatically adjust clock for daylight saving changes" setting
```

Comment:
Probably taking this opportunity, changing the wording/instructions aligned with "Settings app", instead of deprecated "Control Panel" would be good.